Task2/content.css

```diff
@@ -0,0 +1,23 @@
+.content div {
+    margin: 10px;
+    background: rgb(237,238,240);
+    display: inline-block;
+}
```

Comment:
Do you really want that all divs inside `.content` have margin 10px?
I guess that `display: inline-block;` is a redundant. Isn't it?